source/speech/speech.py

```diff
@@ -611,12 +612,7 @@ def speakObject(
 		speak(sequence, priority=priority)
 
 
-
-
-# C901 'getObjectSpeech' is too complex
-# Note: when working on getObjectSpeech, look for opportunities to simplify
-# and move logic out into smaller helper functions.
-def getObjectSpeech(  # noqa: C901

```

Comment:
why was this removed?

Reply:
in fact, where possible, please factor out the new changes into helper function(s)

Reply:
@seanbudd wrote:
> why was this removed?

As written in initial description
> Not directly linked but a a noqa exception for code complexity has been removed because it was not useful anymore.

To be more specific, I have not modified `getObjectSpeech` in this PR but had to reread it while developing. I have figured that `getObjectSpeech` is not complex anymore and thus have removed this `#noqa` directive that was already useless before this PR.

@seanbudd wrote:
> in fact, where possible, please factor out the new changes into helper function(s)

Where do you suggest me to factor code?

Reply:
I was hoping to confirm that the function is under the limits.

> >in fact, where possible, please factor out the new changes into helper function(s)
> 
> Where do you suggest me to factor code?

apologies, I thought the below changes were in the function

Reply:
I have updated initial description to clarify this point.